Lib/test/test_coroutines.py

```diff
@@ -953,11 +953,12 @@ async def b():
 
     def test_corotype_1(self):
         ct = types.CoroutineType
-        self.assertIn('into coroutine', ct.send.__doc__)
-        self.assertIn('inside coroutine', ct.close.__doc__)
-        self.assertIn('in coroutine', ct.throw.__doc__)
-        self.assertIn('of the coroutine', ct.__dict__['__name__'].__doc__)
-        self.assertIn('of the coroutine', ct.__dict__['__qualname__'].__doc__)
+        if not support.MISSING_C_DOCSTRINGS:
+            self.assertIn('into coroutine', ct.send.__doc__)
+            self.assertIn('inside coroutine', ct.close.__doc__)
+            self.assertIn('in coroutine', ct.throw.__doc__)
+            self.assertIn('of the coroutine', ct.__dict__['__name__'].__doc__)
+            self.assertIn('of the coroutine', ct.__dict__['__qualname__'].__doc__)
```

Comment:
Not for this PR to change, but this seems a rather silly test -- it tests that the methods have certain text in their docstrings. I'm curious why that needs to be a test...

Reply:
It was added in 5376ba9630e45ad177150ae68c9712640330a2fc (bpo-24400/#68588) which introduced a distinct type for 'async def' coroutines. I guess it was the simplest way to check that these methods and data descriptors belong to the coroutine type and not inherited from the generator type. It could check the `__objclass__` attribute instead.

I am planning to add signatures for all builtins, so docstrings of coroutine methods will change in the nearest future (they may already be changed in my branch).